setup.py

```diff
@@ -3,6 +3,14 @@
 from setuptools import setup
 
 setup(
-    setup_requires=['pbr>=1.9', 'setuptools>=17.1'],
+    setup_requires=[
```

Comment:
setup_requires is only the requirements to run setup.py, not to install the package. Only pbr and setuptools should be here. pbr then installs the rest from `requirements.txt`. Running `pip install .` installs all requirements. Do you not see this?

Reply:
Not if installed editably with pip install -e .